client/components/SubmissionEmail/SubmissionEmail.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+
+import { Community } from 'types';
+import { communityUrl } from 'utils/canonicalUrls';
+
+type Props = {
+	community: Community;
+	customText: React.ReactNode;
+	submissionTitle: string;
+	submissionUrl?: string;
+	submitterName: React.ReactNode;
+};
+
+const SubmissionEmail = (props: Props) => {
```

Comment:
I imagine we'll eventually use this component to render the text that gets emailed to submitters (and it'll be kept in sync with the preview on the dashboard)